setup.py

```diff
@@ -82,6 +102,7 @@ def run(self):
     ],
     ext_modules=[Extension("swiglpk._swiglpk",
                            sources=["swiglpk/glpk.i"],
+                           swig_opts=["-I"+glpk_header_path],
```

Comment:
Can you try without that? I am pretty sure the include path has to be a directory and not the full path of GLPK.h.